src/common/http.cc

```diff
@@ -595,6 +600,31 @@ ResponseStream::ResponseStream(Message &&other, std::weak_ptr<Tcp::Peer> peer,
   }
 }
 
+ResponseStream &ResponseStream::operator=(ResponseStream &&other) {
+  response_ = std::move(other.response_);
+  peer_ = std::move(other.peer_);
+  buf_ = std::move(other.buf_);
+  transport_ = other.transport_;
+  timeout_ = std::move(other.timeout_);
+
+  return *this;
+}
+
+std::streamsize ResponseStream::write(const char *data, std::streamsize sz) {
+  std::ostream os(&buf_);
+  os << std::hex << sz << crlf;
+  os.write(data, sz);
+  os << crlf;
+  return sz;
+}
+
+std::shared_ptr<Tcp::Peer> ResponseStream::peer() const {
+  if (peer_.expired())
```

Comment:
Please put braces on all single-line blocks inside 'if', 'for'  and 'while' constructs.  I intend to enforce this rule via a linter someday.

Reply:
@dennisjenkins75 I executed `clang-format -i src/common/http.cc` before commit. Ok, I will fix it.